py/htm/encoders/simhash_document_encoder.py

```diff
@@ -0,0 +1,153 @@
+# ------------------------------------------------------------------------------
+# HTM Community Edition of NuPIC
+# Copyright (C) 2016, Numenta, Inc. https://numenta.com
+#               2019, David McDougall
+#               2019, Brev Patterson, Lux Rota LLC, https://luxrota.com
+#
+# This program is free software: you can redistribute it and/or modify it
+# under the terms of the GNU Affero Public License version 3 as published by
+# the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero Public License for
+# more details.
+#
+# You should have received a copy of the GNU Affero Public License along with
+# this program.  If not, see http://www.gnu.org/licenses.
+# ------------------------------------------------------------------------------
+
+import htm.bindings.encoders
+SimHashDocumentEncoder = htm.bindings.encoders.SimHashDocumentEncoder
+SimHashDocumentEncoderParameters = htm.bindings.encoders.SimHashDocumentEncoderParameters
+
+__all__ = ['SimHashDocumentEncoder', 'SimHashDocumentEncoderParameters']
+
+
+if __name__ == '__main__':
+  """
+  Simple program to examine the SimHashDocumentEncoder.
+
+  For help using this program run:
+  $ python -m htm.encoders.simhash_document_encoder --help
+  """
+  import argparse
+  import matplotlib.pyplot as plot
```

Comment:
please make this plot imported only where needed (in the plot, and skip if import fails). So the encoder would run even if not imported.